ui/src/utils.tsx

```diff
@@ -302,6 +302,177 @@ export function getPngMetadata(file: File) {
   });
 }
 
+function parseExifData(exifData) {
```

Comment:
let's put this into a separate pngInfoUtils or imageMetadataUtils since this file is becoming too big